run-tests.sh

```diff
@@ -416,6 +416,16 @@ elif ! (g3kubectl get pods --no-headers -l app=hatchery | grep hatchery) > /dev/
   donot '@exportToWorkspacePortalHatchery'
 fi
 
+#
+# audit-service tests only run for some repos.
+# they don't run for manifest repos because the audit-service might not
+# be deployed everywhere.
+#
+if [[ "$isGen3Release" != "true" && "$service" != "gen3-qa" && "$service" != "fence" && "$service" != "audit-service" && "$service" != "cloud-automation" ]]; then
```

Comment:
Don't we want to make sure we have CI with diff mix-and-match of versions in cdis-manifest PRs?
We should be able to figure out whether audit-service is configured against a given environment by checking the manifest versions block and/or the portal/gitops.json.
Just checking, if you prefer this approach, that's ok.

Reply:
sure